lib/fast_jsonapi/multi_to_json.rb

```diff
@@ -33,7 +38,8 @@ def value!
 
       def rescue
         return self if ok?
-        Result.new { yield(@error) }
+
+        Result.new(*@rescued_exceptions) { yield(@error) }
```

Comment:
Actually the bug was here - after the first `rescue` only `StandardError` was catched.

Reply:
oops :)

```ruby
LoadError.superclass
=> ScriptError
>> LoadError.superclass.superclass
=> Exception

>> StandardError.superclass
=> Exception
```

Reply:
@bf4 oops what? 🙂 Looks like you don't know how `begin-rescue` construction works.

`rescue` catches instances of `StandardError` and its __inheritors__ but NOT __predecessors__.

So, I can't understand why you are talking about `.superclass` 🙁

Reply:
@GarrisonD no, I know how it works. That's why I wrote oops :)

As you pointed out, rescuing StandardError doesn't rescue a LoadError. (That's the relevance of superclass. If LoadError.ancestors.include?(StandardError) there would have been no oops, right?)

Perhaps what wasn't clear was that I wrote the code with the bug and was commenting on *my* oops

Reply:
@bf4 Now everything is clear! Sorry.